src/main/java/uk/gov/hmcts/reform/sscs/json/SscsJsonExtractor.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sscs.json;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SscsJsonExtractor {
+
+    public HashMap<String, Object> extractJson(Map<String, Object> exceptionCaseData) {
+        HashMap<String, Object> pairs = new HashMap<>();
+
+        try {
+            JSONArray jsonArray = new JSONArray(exceptionCaseData.get("scanOCRData").toString());
```

Comment:
what's the structure here?
I'm assuming its not as simple as would be nice,
from what I can tell if it was a nice structure all you would need for this code is:
`return exceptionCaseData.get("scanOCRData")`

Reply:
This is structure received from the callback in CCD so don't think it's possible to do anything nicer. 

```{
  "data": {
    "scanOCRData": [
      {
        "id": "639f82e9-b9d6-4fe7-b34b-94bebb48f755",
        "value": {
          "key": "appellant_first_name",
          "value": "Jack"
        }
      }
    ]
  }
```